engine/app.js

```diff
@@ -52,7 +52,8 @@ const plugins = [
     },
     {
         packagePath: './plugins/ctrl',
-        configPath: path.join(__dirname, 'plugins/ctrl/config.json'),
+        // @TODO: this config have to be removed and build from the symfony box-configuration.
+        config: config.fbs,
```

Comment:
Make sure the correct config is loaded in the tests as well.

https://github.com/bibboxen/bibbox-website/blob/develop/engine/test/state_machine.js#L31
https://github.com/bibboxen/bibbox-website/blob/develop/engine/test/fbs.js#L29
https://github.com/bibboxen/bibbox-website/blob/develop/engine/test/state_machine.js#L31

Reply:
Updated config in tests as well :-)